hircluster.h

```diff
@@ -327,13 +324,24 @@ redisAsyncContext *actx_get_by_node(redisClusterAsyncContext *acc,
                                     redisClusterNode *node);
 
 /* Cluster node iterator functions */
-void initNodeIterator(nodeIterator *iter, redisClusterContext *cc);
-redisClusterNode *nodeNext(nodeIterator *iter);
+void redisClusterInitNodeIterator(redisClusterNodeIterator *iter,
+                                  redisClusterContext *cc);
+redisClusterNode *redisClusterNodeNext(redisClusterNodeIterator *iter);
 
 /* Helper functions */
 unsigned int redisClusterGetSlotByKey(char *key);
 redisClusterNode *redisClusterGetNodeByKey(redisClusterContext *cc, char *key);
 
+/* Deprecated functions and types, kept for backward compatibility. */
```

Comment:
The functions are not deprecated. Only their names. :)

```suggestion
/* Old names of renamed functions and types, kept for backward compatibility. */
```